tests/unit/plugins/inventory/test_kubevirt.py

```diff
@@ -6,10 +6,11 @@
 
 __metaclass__ = type
 
+import copy
```

Comment:
Add newline between? copy is stdlib, pytest is not :)

Reply:
Done.